src/Tizen.NUI/src/public/Layouting/LayoutGroup.cs

```diff
@@ -175,16 +175,7 @@ private void AddChildToLayoutGroup(View child)
                 {
                     // If child of this layout is a pure View then assign it a LayoutGroup
                     // If the child is derived from a View then it may be a legacy or existing container hence will do layouting itself.
-                    if (child.GetType() == typeof(View))
```

Comment:
If you don't check whether child is pure `View` or not, Controls's children set Size and Position twice.
e.g. in case of `Components.Button`
1. `AbsoluteLayout.Onlayout`
2. `Control.OnRelayout` - `Button.OnUpdate` - `UpdateUIContent`

Reply:
Yes, but this also makes something wrong when user just inherit View to make their own component.

class V : View
{
}
....
V myView = new V(); 

myView cannot have LayoutGroup because return value of GetType is V, not View.

And about components thing, we are planning to change inner layout logic to using Layout feature so before that, sizing logic will be called twice but after that, it won't.